flatsurf/geometry/gl2r_orbit_closure.py

```diff
@@ -0,0 +1,940 @@
+# -*- coding: utf-8 -*-
+r"""
+GL(2,R)-orbit closure of translation surfaces
+
+.. TODO::
+
+    - Comparison of renf_elem with Python int not supported (but seem to work
+      through the RealEmbeddedNumberField)
+
+    - the method name ``fromEdge`` is completely misleading
+      (how do we guess that it build a vector, how do we guess that it takes a half-edge as input)
+
+    - Theorem 1.9 of Alex Wright: the field of definition is contained in the field generated by
+      the ratio of circumferences. We should provide a method, .reset_field_of_definition or
+      something similar
+"""
+######################################################################
+#  This file is part of sage-flatsurf.
+#
+#        Copyright (C) 2019-2020 Julian Rüth
+#                      2020      Vincent Delecroix
+#
+#  sage-flatsurf is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  sage-flatsurf is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with sage-flatsurf. If not, see <https://www.gnu.org/licenses/>.
+######################################################################
+
+import cppyy
+import gmpxxyy
+from pyeantic import RealEmbeddedNumberField
+
+import pyflatsurf
+import pyflatsurf.vector
+
+# TODO: move into flatsurf
```

Comment:
What is this one about @videlec

Reply:
I think that I wanted the following to work
```
from pylatsurf import Vertex
```
which is more convenient than
```
Vertex = cppyy.gbl.flatsurf.Vertex
```

Reply:
precised in 8d49414